components/automate-ui/src/app/pages/+compliance/+scanner/containers/jobs-list/jobs-list.component.html

```diff
@@ -23,12 +23,14 @@
   <chef-loading-spinner *ngIf="jobsList.loading" size="100"></chef-loading-spinner>
 
   <chef-tbody *ngIf="!jobsList.loading">
-    <chef-tr class="empty new-row">
-      <span class="cta">Create a new scan job with nodes and profiles</span>
-      <span class="action">
-        <chef-button primary [routerLink]="['/jobs/add']">Create new job</chef-button>
-      </span>
-    </chef-tr>
+    <app-authorized [allOf]="['/compliance/scanner/jobs', 'post']">
+      <chef-tr class="empty new-row">
+        <span class="cta">Create a new scan job with nodes and profiles</span>
+        <span class="action">
+          <chef-button primary [routerLink]="['/jobs/add']">Create new job</chef-button>
```

Comment:
if you dont mind if been trying to get this changed forevar

```suggestion
          <chef-button primary [routerLink]="['/jobs/add']">Create Scan Job</chef-button>
```